drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-client/src/main/java/org/drools/workbench/screens/drltext/client/editor/DRLEditorPresenter.java

```diff
@@ -193,9 +195,15 @@ protected void save(String commitMessage) {
                                                                                        commitMessage);
     }
 
+    @Override
+    protected String getEditorIdentifier() {
+        return EDITOR_ID;
+    }
+
     @OnClose
     public void onClose() {
         versionRecordManager.clear();
+        super.onClose();
```

Comment:
I think we could add `@Override` annotation.

Reply:
Ok, please assume the same comment for whole PR, I mean `onClose` method is updated multiple times.